### PR TITLE
add: `wayland`

### PR DESCRIPTION
**Pull request by Xdavius**
Merge needed for gamescope-git

### CODE REVIEW

**Comment by Xdavius** [comment]
@oklopfer bug ?? Oo

Works on Debian 12 at home

**Comment by Xdavius** [comment]
while installing steam-deb : 

wayland : Casse: libwayland-client0:i386 mais 1.21.0-1 devra être installé
           Casse: libwayland-egl1:i386 mais 1.21.0-1 devra être installé
           Casse: libwayland-server0:i386 mais 1.21.0-1 devra être installé
           Casse: libwayland-cursor0:i386 mais 1.21.0-1 devra être installé

need i386

File: packages/wayland/wayland.pacscript [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+check() {
+  cd "${pkgname}-${pkgver}"
+  meson test -C build --print-errorlogs
+}
+
+package() {
+  cd "${pkgname}-${pkgver}"
+  meson install -C build --destdir "${pkgdir}"
+  mkdir -p docs/share
+  mv "${pkgdir}"/usr/share/{doc,man} docs/share
```
Comment (vigress8) [inline packages/wayland/wayland.pacscript:40]: Why move those?

Comment (Xdavius) [reply packages/wayland/wayland.pacscript:40]: It's a splited package in arch. I want it joined. I have certainly missed something.

**Comment by Xdavius** [comment]
@vigress8 @oklopfer 

Tested rebuild manualy the deb after -IBK.
added multi-arch: same in CONTROL file

```Depends: libglib2.0-0, libffi8, libxml2, libexpat1
Package: wayland
Version: 1.23.0-pacstall1
Architecture: amd64
Section: Pacstall
Priority: optional
Build-Depends: meson, pkg-config, cmake, libxslt1-dev, doxygen, xmlto, graphviz, docbook-xsl, libglib2.0-dev, libffi-dev, libexpat1-dev, libxml2-dev
Provides: libwayland-client0 (= 1.23.0), libwayland-egl1 (= 1.23.0), libwayland-server0 (= 1.23.0), libwayland-cursor0 (= 1.23.0), libwayland-dev (= 1.23.0), libwayland-egl-backend-dev (= 1.23.0), libwayland-doc (= 1.23.0), libwayland-bin (= 1.23.0), wayland
Breaks: libwayland-client0, libwayland-egl1, libwayland-server0, libwayland-cursor0, libwayland-dev, libwayland-egl-backend-dev, libwayland-doc, libwayland-bin
Replaces: libwayland-client0, libwayland-egl1, libwayland-server0, libwayland-cursor0, libwayland-dev, libwayland-egl-backend-dev, libwayland-doc, libwayland-bin
Multi-Arch: same
Homepage: https://wayland.freedesktop.org/
Maintainer: xdavius <xdavius@github.com>
Description: A computer display server protocol
Installed-Size: 2201
```

steam-deb was not broken anymore. All packages in debian got this line, pacstall packages don't have. that's why I have a mess between i386/x64 since a long time I test

I also remove all the -1 pin in preferences.d 

I'm testing using apt-mark hold to force debian not doing weird things. Seems working for now.

File: packages/wayland/wayland.pacscript [hunk 2]
```diff
@@ -0,0 +1,40 @@
+pkgname=wayland
+gives="wayland"
+pkgver=1.23.0
+pkgrel=1
```
Comment (vigress8) [inline packages/wayland/wayland.pacscript:4]:
```suggestion
```
Not needed

**Review by vigress8 — DISMISSED** [review]

**Review by vigress8 — APPROVED** [review]

**Pull request merged**